scipy/linalg/tests/test_solvers.py

```diff
@@ -337,11 +348,10 @@ def _test_factory(case):
         res -= a.conj().T.dot(x.dot(b)).dot(
                     solve(r+b.conj().T.dot(x.dot(b)),b.conj().T).dot(x.dot(a))
                     )
-        assert_array_almost_equal(res,np.zeros_like(res))
-
-    for case in cases:
-        yield _test_factory, case
+        assert_array_almost_equal(res,np.zeros_like(res),decimal=dec)
```

Comment:
added and thank you. 

I saw you work for ASML. This indeed feels like the review Excel files (TRP? forgot its name ) that we used to fill in there :)